### PR TITLE
Added `databricks labs ucx alias` command to create a view of tables from one schema/catalog in another schema/catalog

### LINKED ISSUE #676: Add `databricks labs ucx alias` command
databricks labs ucx alias to create a view between over a table in a different catalog.

### PR DESCRIPTION
**Pull request by FastLee**
### Linked issues
closes #676 

Resolves #..

### Functionality 

- [ ] added relevant user documentation
- [ ] added new CLI command
- [ ] modified existing command: `databricks labs ucx ...`
- [ ] added a new workflow
- [ ] modified existing workflow: `...`
- [ ] added a new table
- [ ] modified existing table: `...`

### Tests

- [ ] manually tested
- [ ] added unit tests
- [ ] added integration tests
- [ ] verified on staging environment (screenshot attached)

### CODE REVIEW

File: src/databricks/labs/ucx/assessment/clusters.py [hunk 1]
```diff
@@ -67,17 +68,21 @@ def _check_init_scripts(self, cluster, failures):
                 continue
             failures.append(f"{_AZURE_SP_CONF_FAILURE_MSG} cluster.")
 
-    def _check_cluster_failures(self, cluster: ClusterDetails):
+    def _check_cluster_failures(self, cluster: ClusterDetails | compute.ClusterSpec):
```
Comment (nfx) [inline src/databricks/labs/ucx/assessment/clusters.py:71]: This is irrelevant to this PR, separate it

File: labs.yml [hunk 2]
```diff
@@ -80,6 +80,23 @@ commands:
       - name: to-schema
         description: target schema to migrate tables to
 
+  - name: alias
```
Comment (nfx) [inline labs.yml:83]: Make PR title match

File: src/databricks/labs/ucx/cli.py [hunk 3]
```diff
@@ -237,6 +237,37 @@ def move(
...
+def alias(
+    w: WorkspaceClient,
+    from_catalog: str,
+    from_schema: str,
+    from_table: str,
+    to_catalog: str,
+    to_schema: str,
+):
+    """move a uc table/tables from one schema to another schema in same or different catalog"""
+    logger.info("Running alias command")
```
Comment (nfx) [inline src/databricks/labs/ucx/cli.py:250]: Why do we need this message to be logged? What users would get from it?

File: src/databricks/labs/ucx/hive_metastore/table_migrate.py [hunk 4]
```diff
@@ -288,6 +385,11 @@ def _recreate_table(self, from_table_name, to_table_name):
         logger.info(f"Creating table {to_table_name}")
         self._backend.execute(create_table_sql)
 
+    def _create_view(self, from_table_name, to_table_name):
```
Comment (nfx) [inline src/databricks/labs/ucx/hive_metastore/table_migrate.py:388]: Rename to create alias view

File: tests/unit/hive_metastore/test_table_move.py [hunk 5]
```diff
@@ -64,10 +67,10 @@ def test_move_tables_not_found_table_error(mocker, caplog):
     assert len([rec.message for rec in caplog.records if "Could not find table SrcC.SrcS.table1" in rec.message]) == 1
 
 
-def test_move_tables_not_found_table_unknown_error(mocker, caplog):
+def test_move_tables_not_found_table_unknown_error(caplog):
     client = create_autospec(WorkspaceClient)
     client.schemas.get.side_effect = [SchemaInfo(), NotFound()]
-    backend = mocker.patch("databricks.labs.ucx.framework.crawlers.StatementExecutionBackend.__init__")
+    backend = backend = create_autospec(StatementExecutionBackend)
```
Comment (nfx) [inline tests/unit/hive_metastore/test_table_move.py:73]: Why assign two times?

**Review by nfx — CHANGES_REQUESTED** [review]

File: src/databricks/labs/ucx/assessment/clusters.py (hunk 1 above)
Comment (FastLee) [reply src/databricks/labs/ucx/assessment/clusters.py:71]: Created PR #839

File: labs.yml (hunk 2 above)
Comment (FastLee) [reply labs.yml:83]: ? it matches

File: src/databricks/labs/ucx/assessment/clusters.py (hunk 1 above)
Comment (FastLee) [reply src/databricks/labs/ucx/assessment/clusters.py:71]: Done

File: src/databricks/labs/ucx/hive_metastore/table_migrate.py (hunk 4 above)
Comment (FastLee) [reply src/databricks/labs/ucx/hive_metastore/table_migrate.py:388]: Done

File: tests/unit/hive_metastore/test_table_move.py (hunk 5 above)
Comment (FastLee) [reply tests/unit/hive_metastore/test_table_move.py:73]: Fixed

**Review by nfx — APPROVED** [review]
Lgtm

**Pull request merged**